frontend/src/hex/HexView.svelte

```diff
@@ -308,107 +302,96 @@
   id="scrollable"
   on:wheel="{(e) => {
     currentPosition += Math.floor(e.deltaY) * alignment;
-    if (currentPosition > dataLength) {
-      currentPosition = dataLength - alignment;
+    if (currentPosition > $dataLength) {
+      currentPosition = $dataLength - alignment;
     }
     if (currentPosition < 0) {
       currentPosition = 0;
     }
   }}"
 >
   <div class="hex-display">
-    {#await dataLenPromise}
-      <LoadingText />
-    {:then dataLength}
```

Comment:
One consequence of this change is that while the new data length thing is loading, it will show outdated information. Probably not actually going to be an issue in practice, but worth thinking about in case it comes up in a bug report.